source/exe/main.cc

```diff
@@ -1,5 +1,9 @@
 #include "exe/main_common.h"
 
+#ifdef WIN32
+#include "exe/service_base.h"
```

Comment:
I am a new hand to envoy, but there seems no `service_base.h` header file in `./source/exe/`. Should it be `#include "exe/win32/service_base.h"`?

Reply:
Are you encountering any issues compiling this or just curious? We use the `strip_prefix` tag on here https://github.com/davinci26/envoy/blob/6650ba0799592927005f59fc00472ce67c03da37/source/exe/BUILD#L196 and this is why `win32` is not needed

Reply:
Just curious about it, thanks a lot for your help : )

Reply:
No prob, happy to help.